RxExample/RxExample/Examples/WikipediaImageSearch/ViewModels/SearchResultViewModel.swift

```diff
@@ -16,12 +16,8 @@ class SearchResultViewModel {
     var title: Observable<String>
     var imageURLs: Observable<[NSURL]>
     
-    var $: SearchViewModel.Dependencies
```

Comment:
I see how removing a layer of boilerplate code could increase readability.

How do you feel about making $ a getter only variable to dependency object.
var $: Dependencies {
     get {
            return Dependencies.sharedDependencies
     }
}

I feel like adding Dependencies.sharedDependencies across the code is duplicating information and adds a lot of visual noise IMHO.